ViewEnvironment.podspec

```diff
@@ -0,0 +1,22 @@
+require_relative('version')
+
+Pod::Spec.new do |s|
+    s.name         = 'ViewEnvironment'
+    s.version      = WORKFLOW_VERSION
+    s.summary      = 'A collection of environment values propagated through a view hierarchy.'
+    s.homepage     = 'https://www.github.com/square/workflow-swift'
+    s.license      = 'Apache License, Version 2.0'
+    s.author       = 'Square'
+    s.source       = { :git => 'https://github.com/square/workflow-swift.git', :tag => "v#{s.version}" }
+
+    # 1.7 is needed for `swift_versions` support
+    s.cocoapods_version = '>= 1.7.0'
+
+    s.swift_versions = ['5.0']
```

Comment:
Nit: move these things into `version.rb` ?

Reply:
Seems reasonable to me—but I'm going to leave it as for this PR to avoid conflating that change here.